frontend/src/components/CustomerStakesVisualization.tsx

```diff
@@ -1,18 +1,79 @@
 import { FC } from 'react';
 import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
 import Tooltip from '@material-ui/core/Tooltip';
 import { Dot } from './Dot';
 import { percent } from '../utils/string';
 import { CustomerStakes } from '../redux/roadmaps/types';
 import css from './CustomerStakesVisualization.module.scss';
+import colors from '../colors.module.scss';
 
 const classes = classNames.bind(css);
 
+const noRatingsStakes = [
+  { id: 1, value: 0.5 },
+  { id: 2, value: 0.3 },
+  { id: 3, value: 0.1 },
+  { id: 4, value: 0.1 },
+];
+
+const StakeVisualizationTooltip: FC<{
+  title: string;
+  children: any;
+}> = ({ title, children }) => (
+  <Tooltip
+    classes={{
+      arrow: classes(css.tooltipArrow),
+      tooltip: classes(css.tooltip),
+    }}
+    title={title}
+    placement="right"
+    arrow
+  >
+    <div>{children}</div>
```

Comment:
Updated the visualization.

This div is required in order for the tooltip to work (somehow it does not get that StakeVisualizationContent starts with div too). I had to be bit more verbose with CustomerStakesVisualization return jsx due to the div breaking css.

Reply:
This seems like a limitation of the tooltip component (https://material-ui.com/components/tooltips/#custom-child-element)

Passing the children directly gives the following warning in the console:
```
index.js:1 Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?

Check the render method of `ForwardRef(Tooltip)`.
```